cmd/demo/main.go

```diff
@@ -15,6 +15,8 @@ func main() {
 	defer w.Destroy()
 	w.SetTitle("Minimal webview example")
 	w.SetSize(800, 600, webview2.HintFixed)
-	w.Navigate("https://en.m.wikipedia.org/wiki/Main_Page")
+	//w.Navigate("https://en.m.wikipedia.org/wiki/Main_Page")
+	//Change website url for now so we have a page with a html select element we can test
+	w.Navigate("https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select")
```

Comment:
```suggestion
	w.Navigate("https://en.m.wikipedia.org/wiki/Main_Page")
```

Reply:
It would be good to revert this change only because I'll send the changes upstream.

Reply:
Makes sense 👍